content/applications/general/voip/voip_widget.rst

```diff
@@ -0,0 +1,76 @@
+===========
+VoIP widget
+===========
+
```

Comment:
a brief sentence or two introducing the VoIP widget could be nice here

Reply:
Noted, I've added this in.